src/pfe/portal/modules/User.js

```diff
@@ -746,8 +747,13 @@ module.exports = class User {
       return registrySecretList;
     }
 
-    const msg = "Failed to patch the Service Account";
+    const msg = "Failed to create the Codewind Secret and/or patch the Service Account, reverting changes to the Docker Config";
```

Comment:
1. Looks like `updateServiceAccountWithDockerRegisrySecret()` will output the failure messages so can this message just be to indicate the changes are being reverted?

2. In `updateServiceAccountWithDockerRegisrySecret()` can we separate the secret POST and the service account PATCH calls into separate blocks so that we can have specific errors for each?

Reply:
done